content/showcase.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Showcase"
+description: ""
+summary: ""
+date: 2025-04-08T02:49:45+02:00
+lastmod: 2025-04-08T02:49:45+02:00
+draft: false
+type: "legal"
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+
+## OpenFusion Tools
+
+### OpenFusion Sites
+
+- [Original / Future Public Server Map Site](https://dexlabs.systems/)
+- [Academy Public Server Map Site](https://academy.dexlabs.systems/)
+
+### GitHub
+
+- [OpenFusionMap](https://github.com/OpenFusionProject/OpenFusionMap)
+- [DexLabs Leaderboards](https://github.com/hichemfantar/dex-labs-leaderboards)
+- [Computress Discord Bot](https://github.com/OpenFusionProject/computress-rs)
+- [Scripts](https://github.com/OpenFusionProject/scripts)
+- [FFSpy](https://github.com/OpenFusionProject/FFSpy)
+- [Disunityff](https://github.com/OpenFusionProject/disunityff)
```

Comment:
Deprecated; ffbuildtool (https://github.com/OpenFusionProject/ffbuildtool) was not made as a replacement, but has the same functionality, so we should at least add ffbuildtool above it